tests/unit/models/test_inboxes.py

```
@@ -0,0 +1,30 @@
+class TestCreateInboxParams:
+    def test_api_data_should_return_dict_with_all_props(self) -> None:
+        entity = CreateInboxParams(name="test")
+        assert entity.api_data == {"name": "test"}
+
+
+class TestUpdateInboxParams:
+    def test_raise_error_when_all_fields_are_missing(self) -> None:
+        with pytest.raises(ValidationError) as exc:
+            _ = UpdateInboxParams()
+
+        assert "At least one field must be provided for update actio" in str(exc)
```

Ruff (0.12.2)

Use of `assert` detected

(S101)